src/openforms/appointments/contrib/qmatic/client.py

```diff
@@ -63,6 +66,19 @@ class BranchDetailDict(TypedDict):
 # API CLIENT IMPLEMENTATIONS, per major version of the API
 
 
+def QmaticClient() -> "Client":
```

Comment:
Feels weird that a class that used to be a `Session` subclass is now a camelcase function. Should this be the `__new__`?

Reply:
It's a factory that pretends to be a class to not break the public API. I think `__new__` is appropriate, I'll give it a try.